src/MOI_wrapper.jl

```diff
@@ -2527,11 +2529,11 @@ anyway.
 """
 function MOI.set(
     model::Optimizer, attr::ConstraintAttribute,
-    ci::MOI.ConstraintIndex{MOI.ScalarAffineFunction{Float64}, <:Any},
+    ci::MOI.ConstraintIndex{MOI.ScalarAffineFunction{Float64}},
     value::T
 ) where T
-    @assert attr.name ∈ keys(CONSTR_ATTR_TYPE) "Unknown constraint attribute for Gurobi backend: $(attr.name)."
-    @assert T <: CONSTR_ATTR_TYPE[attr.name] "Attribute $(attr.name) is a $(CONSTR_ATTR_TYPE[attr.name]) but $T provided."
+    attr.name ∈ keys(CONSTR_ATTR_TYPE) || throw(MOI.UnsupportedAttribute(attr))
```

Comment:
Use `MOI.supports(model, attr, typeof(ci)) || ...`

Reply:
Otherwise this line is not covered

Reply:
Is now covered following  https://github.com/JuliaOpt/Gurobi.jl/pull/243/commits/a6020f9f13aa811c48eb3681259d9986f2f46237